nataliia_romanchuk/homework_8/every_person.py

```diff
@@ -0,0 +1,35 @@
+from datetime import datetime
+
+
+class Person:
+    PREFIX = {"unknown": "Is",
+              "male": "He is",
+              "female": "She is"}
+
+    def __init__(self, first_name, last_name, birth_date, job, working_years, salary, country, city, gender='unknown'):
+        self.first_name = first_name
+        self.last_name = last_name
+        self.birth_date = datetime.strptime(birth_date, "%d.%m.%Y")
+        self.job = job
+        self.working_years = working_years
+        self.salary = salary
+        self.country = country
+        self.city = city
+        self.gender = gender
+
+    def name(self):
+        return f"{self.first_name} {self.last_name}"
+
+    def age(self):
+        now = datetime.now()
```

Comment:
as per requirements there should be specific date for comparrison

Reply:
I didn't get what was the question about. Here I return years return now.year - self.birth_date.year - not_full_year

Reply:
based on requirements:  now is  2018-01-01
but it is ok if it is left as is

Reply:
I prefer to left as it is, due to the lack of time.